src/main/java/org/folio/dataexp/service/export/strategies/RuleFactory.java

```diff
@@ -57,8 +62,43 @@ public RuleFactory(List<Rule> defaultRulesFromConfigFile, List<Rule> defaultHold
     this.defaultHoldingsRulesFromConfigFile = defaultHoldingsRulesFromConfigFile;
   }
 
-
   public List<Rule> getRules(MappingProfile mappingProfile) throws TransformationRuleException {
+    var rules = buildRules(mappingProfile);
+    if (shouldSuppress999ff(mappingProfile)) {
+      rules = rules.stream()
+        .filter(rule -> !is999ff(rule))
```

Comment:
if it is the only place when is999ff method is used it can be refactored to isNot999ff.